app/views/kaui/accounts/_account_info.html.erb

```diff
@@ -18,6 +18,11 @@
                     :class => 'btn btn-xs text-danger',
                     data: {toggle: 'modal', name: @account.name || '(not set)', account_id: @account.account_id}) %>
       <% end %>
+
+      <%= link_to '<i class="fa fa-arrow-down" aria-hidden="true"></i>'.html_safe,
```

Comment:
Need a permission check. I think something like `if can? :export, Kaui::Admin` might work (https://github.com/killbill/killbill-api/blob/7ea823b25e8fd6299350ec62d700e265022be34e/src/main/java/org/killbill/billing/security/Permission.java#L114C41-L114C56). To be tested...